src/Command/DumpRepositoriesCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OAT\DependencyResolver\Command;
+
+use OAT\DependencyResolver\Repository\RepositoryMapConverter;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DumpRepositoriesCommand extends Command
+{
+    /** @var RepositoryMapConverter */
+    private $repositoryMapConverter;
+
+    public function __construct(RepositoryMapConverter $repositoryMapConverter)
+    {
+        parent::__construct();
+
+        $this->repositoryMapConverter = $repositoryMapConverter;
+    }
+
+    protected function configure()
+    {
+        $this->setName('repositories:dump')
+            ->addOption(
+                'filename',
```

Comment:
For me this is misleading, because you expect file path here. `dirname` function also expects a path, not a single filename.